cli_tools/gce_vm_image_import/importer/bootable_disk_processor.go

```diff
@@ -89,6 +89,10 @@ func newBootableDiskProcessor(args ImportArguments, pd persistentDisk) (processo
 		return nil, err
 	}
 
+	// Temporary fix to ensure gcloud shows daisy's output.
+	// A less fragile approach is tracked in b/161567644.
+	workflow.Name = LogPrefix
```

Comment:
LogPrefix can be a field in ImportArguments instead of a constant. Since this constant is in a different file, could be hard to track.

Reply:
Were you thinking of [this struct](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/cli_tools/gce_vm_image_import/importer/args.go#L45)? Was the thought to have the prefix be settable by a user?

The long-term fix (which I'd like to do sooner rather than later) will remove this global constant, and instead pass a pre-configured logger to daisy, thus removing this coupling between workflow names and gcloud's filter.

Reply:
If this change is temporary, you can leave the constant. I'm just not a big fan of cross-file constants in a single package as it's hard to track it outside an IDE.